libs/common-hooks/src/useAvailableChains.ts

```diff
@@ -0,0 +1,15 @@
+import { useMemo } from 'react'
+
+import { getAvailableChains } from '@cowprotocol/common-utils'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+
+import { useFeatureFlags } from './index'
+
+export function useAvailableChains(): SupportedChainId[] {
+  const { isArbitrumOneEnabled } = useFeatureFlags()
+
+  return useMemo(
+    () => getAvailableChains(isArbitrumOneEnabled ? undefined : [SupportedChainId.ARBITRUM_ONE]),
+    [isArbitrumOneEnabled]
```

Comment:
From now on, that's all we need to change to hide a `SupportedChainId` from the UI